oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/ScopeDefaultColumn.java

```diff
@@ -34,14 +34,16 @@ public class ScopeDefaultColumn {
     private String columnName;
     private Class<?> type;
     private boolean isID;
+    private boolean banyandbNoIndexing;
     private int length;
 
-    public ScopeDefaultColumn(String fieldName, String columnName, Class<?> type, boolean isID, int length) {
+    public ScopeDefaultColumn(String fieldName, String columnName, Class<?> type, boolean isID, int length, boolean banyandbNoIndexing) {
```

Comment:
This seems an unreasonable change. `ScopeDefaultColumn` is never storage related.

Reply:
`service_id` is defined by `ScopeDefaultColumn`.  IIUC, `ScopeDefaultColumn.columnName` defines the column name in the storage layer. Essentially, I tend to mark `service_id` as a non-indexed tag. Adding a new field seems relevant.

Reply:
`isID` is storage-related, right?

Reply:
Dropped it.